src/EPR.Payment.Service/Program.cs

```diff
@@ -101,19 +101,19 @@
 var featureManager = app.Services.GetRequiredService<IFeatureManager>();
 var logger = app.Services.GetRequiredService<ILogger<Program>>();
 
-bool enablePaymentsFeature = await featureManager.IsEnabledAsync("EnablePaymentsFeature");
-bool enablePaymentStatusInsert = await featureManager.IsEnabledAsync("EnablePaymentStatusInsert");
-bool enablePaymentStatusUpdate = await featureManager.IsEnabledAsync("EnablePaymentStatusUpdate");
-bool enableGetPaymentByExternalPaymentId = await featureManager.IsEnabledAsync("EnableGetPaymentByExternalPaymentId");
+bool enableOnlinePaymentsFeature = await featureManager.IsEnabledAsync("EnableOnlinePaymentsFeature");
```

Comment:
sonarqube: $ contains different logging templates which is a problem for logging. So instead of interpolatoin, pass parameters separately. "The logging message template should not vary between calls to 'LoggerExtensions.LogInformation(ILogger, string?, params object?[])"

instead of
logger.LogInformation($"EnableOnlinePaymentsFeature: {enableOnlinePaymentsFeature}");

do
logger.LogInformation("EnableOnlinePaymentsFeature: {EnableOnlinePaymentsFeature}", enableOnlinePaymentsFeature);

Do this for all the 4 feature flags logging